config/feeds_config_v2.json

```
@@ -52856,8 +52856,8 @@
     },
     {
       "id": "100001",
-      "full_name": "ynETH MAX (ynETHx) - YieldNest convertToAssets on ETH",
-      "description": "YieldNest convertToAssets call on ETH mainnet at 0x657d9ABA1DBb59e53f9F3eCAA878447dCfC96dCb",
+      "full_name": "ynETHx / WETH",
+      "description": "Yieldnest Restaked Ethereum / Wrapped Ethereum Redemption Rate. YieldNest convertToAssets call on ETH mainnet at 0x657d9ABA1DBb59e53f9F3eCAA878447dCfC96dCb",
```

nit: `Yieldnest ` should be with capitalized "N" - `YieldNest`
```suggestion
      "description": "YieldNest Restaked Ethereum / Wrapped Ethereum Redemption Rate. YieldNest convertToAssets call on ETH mainnet at 0x657d9ABA1DBb59e53f9F3eCAA878447dCfC96dCb",
```